src/tlog-test-json-overlay.c

```diff
@@ -29,7 +29,8 @@
 #include <assert.h>
 
 bool
-test(const char *name,
+test(const char *file, int line,
```

Comment:
Could you please put `file` and `line` arguments on separate lines to keep to the way other arguments are listed? This way it will be less likely `line` is overlooked.